content/en/docs/v3.6/op-guide/kubernetes.md

```diff
@@ -0,0 +1,438 @@
+---
+title: Run etcd clusters as a Kubernetes StatefulSet
+weight: 4200
+description: Running etcd as a Kubernetes StatefulSet
+---
+
+Below demonstrates how to perform the [static bootstrap process](../clustering/#static) as a Kubernetes StatefulSet.
+
+# Example Manifest
+This manifest contains a service and statefulset for deploying a static etcd cluster in kubernetes.
+
+If you copy the contents of the manifest into a file named `etcd.yaml`, it can be applied to a cluster with this command.
+```
```

Comment:
Markdownlint requires fenced code blocks to have blank lines around them and have a language specified. Can you please update through the page and check.
```suggestion

```bash
```